roundware/rw/models.py

```diff
@@ -624,14 +624,15 @@ def __unicode__(self):
 
 
 class Vote(models.Model):
+    voter = models.ForeignKey(settings.AUTH_USER_MODEL, null=True, blank=True)
     value = models.IntegerField(null=True, blank=True)
     session = models.ForeignKey(Session)
     asset = models.ForeignKey(Asset)
     type = models.CharField(
-        max_length=16, choices=[('like', 'like'), ('flag', 'flag'), ('rate', 'rate')])
+        max_length=16, choices=[('like', 'like'), ('flag', 'flag'), ('rate', 'rate'), ('block_asset', 'block_asset'), ('block_user', 'block_user')])
```

Comment:
I prefer storing the choices as a class variable. This way, I can retrieve the options elsewhere in the code.

```
class Vote:
    TYPES = [('like', 'like'), ('flag', 'flag'), ('rate', 'rate'), ('block_asset', 'block_asset'), ('block_user', 'block_user')]
    type = models.CharField(max_length=16, choices=TYPES)
```

Then you can refer to Vote.TYPES if you want to use it elsewhere. See https://docs.djangoproject.com/en/1.10/ref/models/instances/#django.db.models.Model.get_FOO_display